src/components/PrincipalSection/index.tsx

```diff
@@ -1,10 +1,52 @@
+import Diablo4 from '~/../../public/assets/logos/diablo-4.png'
+import HearthStone from '~/../../public/assets/logos/hearthstone.png'
+import WoW from '~/../../public/assets/logos/world-of-warcraft.png'
+import Diablo1 from '~/../../public/assets/logos/diablo-1.png'
+import StarCraft2 from '~/../../public/assets/logos/star-craft-2.png'
+import Diablo4XL from '~/../../public/assets/logos/diablo-4xl.png'
+import ThumbDiablo4 from '~/../../public/assets/banner/thumb-diablo4.png'
+
 import { Navbar } from 'components'
 
 const PrincipalSection = () => {
   return (
     <div className="min-h-[738px] bg-1 ">
       <Navbar />
-      <div></div>
+      <div className="flex min-h-[738px] items-center justify-center">
```

Comment:
The second one is for the texts/buttons/images, because I didn't have a size to centralize the elements in the vertically mid, so I thought if I put the same height as the background, I'II able to centralize the elements because the background height length is the same as the div length.

Reply:
The main problem is that the first `min-h-[738px]` is completely unnecessary because you already have a child that has that same minimum height. You could use `flex-grow` in that child `div` to make it expand to use all available space that was created by the minimum height rule.